.github/workflows/jekyll.yml

```
@@ -1,20 +1,76 @@
 name: Jekyll site CI
-
 on:
-  push:
-    branches: [ changes, release, translate ]
   pull_request:
-    branches: [ changes, release, translate ]
-
+    branches: [ next-release, release ]
+  push:
+    branches: [ next-release ]
 jobs:
-  build:
+  Jekyll_site_CI:
```

```suggestion
  jekyll_site_ci:
```

Personal preference.